### PR TITLE
Add Wearable slider example for Tizen 5.5 TAU documentation

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU-Design-Editor/issues/447
[Problem] No wearabale slider example link for 5.5 documentation
[Solution] Update documentation

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
@mijin85cho please don't merge yet, we are waiting for final approval

**Comment by lslachciak** [comment]
I rebased PR basing on the previous depending PRs

**Comment by mijin85cho** [comment]
To rebuild the staging link, I close and reopen this PR.

**Comment by lslachciak** [comment]
added minor commit to this PR removing double image in wearable documentaiton

**Comment by lslachciak** [comment]
I rebased my PR over latest @singa2000 changes

**Comment by lslachciak** [comment]
we need to improve https://docs.stg.tizen.org/staging/959/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm

**Comment by lslachciak** [comment]
rebased with the fix for  https://docs.stg.tizen.org/staging/959/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm
-rmeoved unneded refernce to video

**Comment by singa2000** [comment]
We get the verification PASS.
Would you merge it, please?

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm [hunk 1]
```diff
@@ -24,8 +24,20 @@ <h1>Section Changer</h1>
 	</tr>
 	</tbody>
 </table>
+<<<<<<< HEAD
```
Comment (mijin85cho) [inline docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm]: Please remove these conflict marks.

image

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm [hunk 2]
```diff
@@ -14,6 +14,13 @@ <h1>Circular Index Scroll Bar</h1>
 <p>If you spin the rotary quickly, an indicator will be displayed and you can move in an index unit.</p>
 <p>The indicator will disappear if no rotary action is detected for a second.</p>
 <p>This component fires a select event when the index characters are selected.</p>
+<button
```
Comment (mijin85cho) [inline docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm]: Just for check.
Unlike other pages, in this page, the image location is under the note.
Is it intended?
image

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm [hunk 3]
```diff
@@ -24,6 +24,7 @@ <h1>Button</h1>
 >Use example</button>
 
 <p>The following table describes the supported button classes.</p>
+<img src="../../images/button.png" alt="photo of button element in Tizen">
```
[the remark this answers is not in the document]
Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm]: Issue confirmed.

File: docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm (hunk 1 above)
Comment (lslachciak) [reply docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm]: conflicts removed

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm (hunk 2 above)
Comment (lslachciak) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm]: improved order

**Comment by lslachciak** [comment]
stg urls: 
https://docs.stg.tizen.org/staging/959/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm
https://docs.stg.tizen.org/staging/959/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_SectionChanger.htm
https://docs.stg.tizen.org/staging/959/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm
https://docs.stg.tizen.org/staging/959/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm (hunk 3 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm]: I can see other issues are fixed, but this one is not.
We still have same two images in this page. Would you please check?

**Comment by lslachciak** [comment]
@mijin85cho could you check my last comment?

**Comment by mijin85cho** [comment]
> @mijin85cho could you check my last comment?

I think [this comment](https://github.com/Samsung/tizen-docs/pull/959#discussion_r361346505) is attached after yours...

**Comment by lslachciak** [comment]
@mijin85cho you are right, I missed it,
I fixed PR according to your comment, please check stg url:
https://docs.stg.tizen.org/staging/959/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm

**Comment by mijin85cho** [comment]
> @mijin85cho you are right, I missed it,
> I fixed PR according to your comment, please check stg url:
> https://docs.stg.tizen.org/staging/959/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm

Thanks for the update. I could see the issue is fixed.

@TomaszLukawskiSam  Would you please re-approve this PR with the latest version?

**Comment by mijin85cho** [comment]
Thanks for the update and check.

**Pull request merged**